profondo/cnn.py

```diff
@@ -28,26 +29,35 @@ def load_data(path):
     return meta, images
 
 
-def train_test_split(meta, images, train_ratio=0.8):
+def train_test_split(meta, images, train_ratio=0.8, validation_ratio=0.2):
     idx = np.random.permutation(np.arange(len(images)))
```

Comment:
Let's set the seed for the random number generator so that if the training is re-run we keep using the same training data.